sdk/python/tests/compiler/compiler_withparams_test_helper.py

```diff
@@ -30,8 +30,7 @@ def pipeline():
                 name="my-out-cop0",
                 image='python:alpine3.6',
                 command=["sh", "-c"],
-                # arguments=['python -c "import json; import sys; json.dump([i for i in range(20, 31)], open(\'/tmp/out.json\', \'w\'))"'],
-                arguments=['python -c "import json; import sys; json.dump([{\'a\': 1, \'b\': 2}, {\'a\': 10, \'b\': 20}], open(\'/tmp/out.json\', \'w\'))"'],
+                    arguments=['python -c "import json; import sys; json.dump([{\'a\': 1, \'b\': 2}, {\'a\': 10, \'b\': 20}], open(\'/tmp/out.json\', \'w\'))"'],
```

Comment:
Stray change?

Reply:
test dev scratchpad.  i left in in there because it'll be useful whenever we go back to iterate on this featrue